### ISSUE TITLE
pre-train internally

### ISSUE THREAD

**Issue by giacbrd** [giacbrd/ShallowLearn#9]
Given a training set, it is possible to train the word embeddings (unsupervised) first, and then doing the label learning. It is similar to pass a pre-trained model, but more transparent

**Comment by giacbrd** [comment]
added fit_embeddings(X) to GensimFastText in https://github.com/giacbrd/ShallowLearn/commit/09c25aae14a308bc817941d4e85044634118693e
